client/src/components/User/__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -5,15 +5,20 @@ exports[`<User /> should render component 1`] = `
   className="user_container"
 >
   <div
-    className="user-avatar-container"
+    className="avatar"
   >
     <div
-      className="avatar-picture"
+      className="user-avatar-container fit-parent"
     >
-      <img
-        alt="avatar"
-        src="/images/avatar.png"
-      />
+      <div
```

Comment:
Avatar should be mocked. Changes in avatar should not affect snapshot of the User component